db/seeds.rb

```diff
@@ -7,3 +7,7 @@
 #
 #   movies = Movie.create([{ name: 'Star Wars' }, { name: 'Lord of the Rings' }])
 #   Character.create(name: 'Luke', movie: movies.first)
+if Norm.count.zero?
+  Norm.create(name: 'full_time')
+  Norm.create(name: 'part_time')
```

Comment:
Here's a way to write this that involves adding less code when more norms are added:

```ruby
%w[full_time part_time].each { |norm| Norm.create(norm) }
```

Reply:
thans :)